base/applications/network/ping/ping.c

```diff
@@ -425,14 +428,20 @@ Ping(void)
 
     if (RequestSize != 0)
     {
-        SendBuffer = malloc(RequestSize);
+        SendBuffer = (PUCHAR)malloc(RequestSize);
         if (SendBuffer == NULL)
         {
             ConResPrintf(StdErr, IDS_NO_RESOURCES);
             exit(1);
         }
 
-        ZeroMemory(SendBuffer, RequestSize);
+        // Windows ping utility fills the optional data field
+        // with ASCII characters from 'a' to 'w' wrapping back around
+        // until SendBuffer is full
+        for(int i = 0; i  < RequestSize; i++)
+        {
+            ((PUCHAR)SendBuffer)[i] = (UCHAR)(ALPHABET_START + (i % ALPHABET_LENGTH));
```

Comment:
```suggestion
            ((PUCHAR)SendBuffer)[i] = (UCHAR)('a' + (i % ('w'-'a'+1)));
```
Much more self-documented than the macros which themselves used "magic" numbers.